emf/loadflow_tool/helper.py

```diff
@@ -309,4 +310,26 @@ def get_metadata_from_filename(file_name):
             file_metadata["Model.domain"],\
             file_metadata["Model.forEntity"] = entity_and_area_list
 
-    return file_metadata
\ No newline at end of file
+    return file_metadata
+
+
+def export_model(network: pypowsybl.network, CGM_meta, profiles=None):
```

Comment:
1. Rename CGM_meta to opdm_object_meta
2. Get base filename only once, in the beginning of the function